src/ipc-rpc-state-share.ts

```diff
@@ -109,7 +109,7 @@ export abstract class RpcServer {
 
       response = { status: 200, ...await this.handleRpc(content) };
     } catch (err) {
-      response = { status: 500, error: err && typeof err === 'object' && 'message' in err ? err.message : String(err) };
+      response = { status: 500, error: err && typeof err === 'object' && 'message' in err ? (err as Error).message : String(err) };
```

Comment:
Actually, why this change? We’re already explicitly checking whether `err.message` exists on this line

Reply:
It doesn’t in CI and it doesn’t locally for me, there might just be something wrong with your local setup? (But even then, TS _should_ see the `'message' in err` check, which should prevent it from presenting this type of error message)

Reply:
Tried to bump typescript package from 4.5.4 to 5.1.3 and the error is gone. I wonder what was your version when you tried locally.

Reply:
Mh yeah, we might want to bump the minimum `typescript` version in `package.json`, although you should usually get that anyway on a fresh checkout because we’re not checking in the `package-lock.json` file